README.md

```diff
@@ -1,4 +1,8 @@
 # ibmcloud-storage-volume-lib
+
+[![Build Status](https://travis.ibm.com/IBM/ibmcloud-storage-volume-lib.svg?token=ptbMq1PWtBZJZhvq3cph)](https://travis.ibm.com/IBM/ibmcloud-storage-volume-lib)
```

Comment:
Is it working? as its internal to IBM and our repo is open source

Reply:
Need to work on it.

I am going to work on it.